guildwars2/misc.py

```diff
@@ -11,14 +11,41 @@
 class MiscMixin:
     @commands.command(aliases=["gw2wiki"])
     async def wiki(self, ctx, *, search):
-        """Search the Guild wars 2 wiki"""
+        """Search the Guild wars 2 wiki
+        
+        Append a 2-character language tag to search in the localized wikis
+        Available languages:
+        en
+        de
+        fr
+        es
+        
+        Defaults to en
+
+        Examples:
+        Search for Lion's Arch:
+        $wiki Lion's Arch
+        Search it in the german wiki:
+        $wiki Löwenstein de
+        """
         if len(search) > 300:
             await ctx.send("Search too long")
             return
-        wiki = "https://wiki.guildwars2.com"
+        wiki = {"en": "https://wiki.guildwars2.com",
+            "de": "https://wiki-de.guildwars2.com",
+            "fr": "https://wiki-fr.guildwars2.com",
+            "es": "https://wiki-es.guildwars2.com"}
+        searchurl = {"en": "{}/index.php?title=Special%3ASearch&search={}",
+            "de": "{}/index.php?search={}&title=Spezial%3ASuche&",
+            "fr": "{}/index.php?search={}&title=Spécial%3ARecherche",
+            "es": "{}/index.php?title=Especial%3ABuscar&search={}"}
+        lang = search.split(" ")[-1]
+        if lang in wiki:
```

Comment:
lang.lower()` would be better perhaps as to make langauge parameter case insensitive